docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Nebula docs
+---
+
+**Nebula** is a distributed, scalable, lightning-fast graph database. It is the only solution in the world capable to host graphs with dozens of billions of vertices (nodes) and trillions of edges, while still provides millisecond latency.
+
+**Nebula's** goal is to provide reading, writing, and computing with high concurrency, low latency for super large scale graphs. Nebula is an open source project and we are looking forward to working with the community to popularize and promote the graph database.
+
+As a graph database, **Nebula** has these features
+* Symmetrically distributed
+* Highly scalable
+* Fault tolerant
+* Strong data consistency
+* SQL-like query language
+
+## How can I get Nebula ##
+**Nebula** source code is available here on [`GitHub`](https://github.com/vesoft-inc/nebula). The currently release is version 0.1. Please refer the `Release Notes` here for details. You can also download [`docker` ](https://www.docker.com/get-started)image to try it.
+More details on how to get Nebula image click [`Get Started`](https://github.com/vesoft-inc/nebula/blob/master/docs/get-started.md).
```

Comment:
I think get-started.md is not just about how to get Nebula image, it is much more like the _Quick Start_